reference/changelog.md

```diff
@@ -58,6 +63,98 @@ redirect_from: /install/changelog.html
 
 > See [known issues](../reference/known-issues.html) for currently known issues and workarounds.
 
+## v0.15.1 {#v151}
+
+Released on September 11, 2018
+
+In addition to the 0.15.1 CLI release, we've released new versions of all of our packages.
+
+### Pulumi CLI
+
+Major features of this release include:
+
+#### Ephemeral status messages
+
+Providers are now able to register "ephmeral" update messages which are shown in the "Info" column in the CLI during an update, but which are not printed at the end of the update. The new version of the `@pulumi/kubernetes` package uses this when printing messages about resource initialization.
+
+#### Local backend
+
+The local backend (which stores your deployment's state file locally, instead of on pulumi.com) has been improved. You can now use `pulumi login --local` or `pulumi login file://<path-to-storage-root>` to select the local backend and control where state files are stored. In addition, older versions of the CLI would behave slightly differently when using the local backend vs pulumi.com, for example, some operations would not show previews before running.  This has been fixed.  When using the local backend, updates print the on disk location of the checkpoint file that was written.
```

Comment:
Link to new page describing how this works in more detail? "To learn more, ..."

Reply:
Thanks! Added a link to the state and backend page.